certificate-manager/src/main/java/io/strimzi/certs/OpenSslCertManager.java

```diff
@@ -389,7 +393,7 @@ private void exec(List<String> cmd) throws IOException {
             outputStream.close();
 
             int result = proc.waitFor();
-            String stdout = new String(Files.readAllBytes(out.toPath()), Charset.defaultCharset());
+            String stdout = Files.readString(out.toPath(), Charset.defaultCharset());
```

Comment:
We might not need these changes in `OpenSslCertManager` now that we're not implementing a conversion webhook.

Reply:
This is just a simple change, as suggested by IDE.

Reply:
Can we drop this to avoid conflicts? I'm working in this area at the moment and expect this class to get refactored.